ledger/src/snapshot_utils.rs

```diff
@@ -30,7 +30,13 @@ pub const TAR_SNAPSHOTS_DIR: &str = "snapshots";
 pub const TAR_ACCOUNTS_DIR: &str = "accounts";
 pub const TAR_VERSION_FILE: &str = "version";
 
-pub const SNAPSHOT_VERSION: &str = "1.0.0";
+pub const SNAPSHOT_VERSION_1_0: &str = "1.0.0";
+pub const SNAPSHOT_VERSION: &str = "1.1.0";
```

Comment:
Oh on `master` you'll need to bump SNAPSHOT_VERSION to 1.10 as well

Reply:
Yup I can add that to my other PR for master!